### ISSUE TITLE
Tensorboard graphs are a mess when using with keras

### ISSUE THREAD

**Issue by dickreuter** [fchollet/keras#7465]
When I create a simple Keras Model

    model = Sequential()
    model.add(Dense(10, activation='tanh', input_dim=1))
    model.add(Dense(1, activation='linear'))
    model.compile(loss='mean_squared_error', optimizer='adam', metrics=['mean_squared_error'])

and do a callback to Tensorboard
    
    tensorboard = TensorBoard(log_dir='c:/temp/tensorboard/run1', histogram_freq=1, write_graph=True, write_images=False)
    model.fit(x, y, epochs=1000, batch_size=1, callbacks=[tensorboard])

The output in Tensorboard looks like this:
![5nfjw](https://user-images.githubusercontent.com/1256318/28739001-0e190492-73ef-11e7-9f24-30d85d036e43.png)

In other words, it's not well structured, probably because the items are not labeled correctly.

Is there anything I can do to make the graphs output look more structured?
